b_server.md

```diff
@@ -262,86 +393,726 @@ If the session exists and has been destroyed returns true.
+      },
+      "created": "2018-11-07T20:53:14.322Z",
+      "rights_holder_public_key": null,
+      "used_at": null,
+      "invite_url": "http://a-portal.jlinc.test/regauth/gbobcog08300400501981586g0667841d8babb10ae98f4ba2426f75f2"
+    },
+    {
+      "email": "user2@example.com",
+      "invite_token": "0667841d8babb10ae98f4ba2426n75f2",
+      "organization_apikey": "bobco",
+      "external_id__c": "jlinc-s33xHBpBrc59kh-rSpVQPBMgMKzyHQTe",
+      "personal_data": {
+        "email": "user1@example.com",
+        "firstname": "carol",
+        "lastname": "lorac",
+        "homephone": "123-234-3456",
+        "mailingcity": "Oakland",
+        "mobilephone": "111-111-2222",
+        "mailingstate": "CA",
+        "mailingstreet": "121 Main Street",
+        "mailingcountry": "US",
+        "mailingpostalcode": "01237"
+      },
+      "created": "2018-11-07T20:53:14.322Z",
+      "rights_holder_public_key": null,
+      "used_at": null,
+      "invite_url": "http://a-portal.jlinc.test/regauth/gbobcog08300400501981586g1b8df697e27dce7b08dbae96cd69e1b2"
+    }
+  ],
+  "success": true
+}
+```
 
 {% include starsep.html %}
 
-### Get all the end-user account data for an org
+<a name="delete-end-user-invite"></a>
+### Deleting an end-user invite
+
+Deletes the end-user invite matching the given `inviteToken`.
+
+_Endpoint:_
+```
+POST http://b-api.jlinc.test/api/delete_end_user_invite
+```
+
+_Post body:_
+```json
+{
+  "inviteToken": "08300400501981586g1b8df697e27dce7b08dbae96cd69e1b2"
+}
+```
+
+If a matching invite exists and has been deleted.
+
+_Response:_
+```json
+{
+  "success": true
+}
+```
+
+{% include starsep.html %}
 
-_GET https://sandbox.b.jlinclabs.net/api/end_user_account_data/{session_id}
+<a name="get-all-end-user-account-data"></a>
+### Getting all end-user account data for an organization
 
-Requires a valid org admin session id.
+_Endpoint:_
+```
+GET http://b-api.jlinc.test/api/end_user_account_data
+```
 
-Returns an array of end-user account data.
+Returns an array of all end-user's account data that the end-user has shared with the organization.
 
+_Response:_
 ```json
 {
- "success": true,
- "allEndUserAccountData": ["..."]
+    "allEndUserAccountData": [
+        {
+            "rhldr_id": "W_puM0eb-iRRvhUcKNUE9wcDCokSD6IHk5-FO0p-GJM",
+            "personal_data": {
+                "email": "al.pacino@example.com"
+            },
+            "shared_personal_data": {},
+            "consents": {},
+            "communication_channels": {}
+        },
+        {
+            "rhldr_id": "fDry8k1Y7J0uZ1Tnkmb6UtPqBDkvRiVOcSqYHfULr-M",
+            "personal_data": {
+                "email": "alice.walker@example.com",
+                "firstname": "alice",
+                "lastname": "walker",
+                "gender": "unique",
+            },
+            "shared_personal_data": {...},
+            "consents": {...},
+            "communication_channels": {...}
+        }
+    ],
+    "success": true
```

Comment:
Put all the `"success": true` at the top of the JSON examples in this __entire document__